violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/EditorPart.java

```diff
@@ -47,7 +49,28 @@
  */
 public class EditorPart extends JPanel implements IEditorPart
 {
-
+    private IGraph graph;
+    
+	private IGrid grid;
+	
+	private double zoom;
+	
+	private IEditorPartSelectionHandler selectionHandler = new EditorPartSelectionHandler();
+	
+	private int lastWidth = 0;
+	
+	private int lastHeight = 0;
+	
+	/**
+	 * Scale factor used to grow drawing area
+     */
+	private static final double GROW_SCALE_FACTOR = Math.sqrt(2);
+	
+	private IEditorPartBehaviorManager behaviorManager = new EditorPartBehaviorManager();
+	
+    private String deleteDialogText = "Delete selected elements?";
```

Comment:
Why is this hard-coded ? You should use properties for all languages available in VioletUML